### ISSUE TITLE
Are the following packages still necessary in a new project?

### ISSUE THREAD

**Issue by toddw** [TrueCar/gluestick#956]
In GlueStick 0.10.x we needed to make sure a project had the same packages as GlueStick. Now that `gluestick` itself is a dependency I assume this is no longer the case. Is this the true? Are these packages really necessary for GlueStick to work? 

```
babel-plugin-gluestick
css-loader
extract-text-webpack-plugin
image-webpack-loader
sass-loader
style-loader
```

**Comment by zamotany** [comment]
Some of the dependencies in `gluestick` should be `peerDependencies`, so it will be up to developer to decide which exact version of library they want to use. This includes babel stuff, loader, react stuff and redux stuff. In next release `gluestick` will have those in `peerDependencies`, then `auto-upgrade` will be used to update few files created by gluestick and to check if it's `peerDependencies` are met. There are few reasons as why to use this approach, firstly it reduces possibility of having duplicated version of the same library in bundle. Secondly, updating project dependencies is easier that the `gluestick` ones, since it requires to follow publishing procedure, run all tests, e2e, write changelog and so on.

**Comment by zamotany** [comment]
Closed as this issue is irrelevant, due to #991
